src/base/emu-i386/simx86/trees.c

```diff
@@ -970,6 +971,8 @@ TNode *Move2Tree(IMeta *I0, CodeBuf *GenCodeBuf)
 	}
 	/* ->REPLACE the code of the node found with the latest
 	   compiled version */
+	if (nG->alive > 0)
```

Comment:
I am not sure I understand the logic here.
How can we find the alive node, if we already
checked querymarks to avoid any possible
overlap?

Reply:
I think you are correct and I can remove this unmark.

I'll double-check the logs and will let you know later: there was a clear case of replacing a node being buggy but I think it was an aged node.

Reply:
I now double-checked that the buggy behaviour was only because of decayed nodes not unmarking their memory.